Library.py

```diff
@@ -209,71 +258,96 @@ def return_book():
-        if confirm == "Y":
+        confirm = input("\nIs the information correct? [Y/n]: ").strip().upper()
+        if confirm != "N":
             # Remove the book from the database
-            cursor.execute('DELETE FROM books WHERE title = ? COLLATE NOCASE', (title,))
+            cursor.execute("DELETE FROM books WHERE title = ? COLLATE NOCASE", (title,))
             conn.commit()
-            print(f"Book '{title}' has been removed from the library.")
+            log("info", f"Book '{title}' has been removed from the library.")
             formatting()
         else:
             print("\nBook removal canceled.")
 
     conn.close()
 
 
+def has_books_table():
+    with sqlite3.connect("library.db") as conn:
+        cursor = conn.cursor()
+        cursor.execute("SELECT * FROM books")
+        try:
+            cursor.fetchall()
+            return True
+        except sqlite3.OperationalError:
+            return False
+
+
 # Interactive menu
 def menu():
     while True:
         formatting()
         print("Main Menu")
         formatting()
         sleep(1)
-        print("Options:")
-        print("[1] List Registered Books.\n"
-              "[2] Register a New Book.\n"
-              "[3] Search the Library.\n"
-              "[4] Borrow a Book.\n"
-              "[5] Return a Book.\n"
-              "[6] Remove a Book from the Library.\n"
-              "[0] Exit the Program.")
-        formatting()
-        try:
-            choice = int(input("Please type your choice: "))
-        except KeyboardInterrupt:
-            print("\033[31mInterruption detected. Please restart the program.\033[m")
-            break
-        except (ValueError, TypeError):
-            print("\033[31mInvalid input. Please choose a valid option.\033[m")
-        if choice not in range(0, 7):
-            print("\033[31mInvalid input. Please choose a valid option.\033[m\n")
-        if choice == 0:
+
+        if not has_books_table():
+            # If table doesn't exist there is no other option other than add books
+            choice = 2
+        else:
+            print("Options:")
+            print(
+                "[1] List Registered Books.\n"
+                "[2] Register a New Book.\n"
+                "[3] Search the Library.\n"
+                "[4] Borrow a Book.\n"
+                "[5] Return a Book.\n"
+                "[6] Remove a Book from the Library.\n"
+                "[0] Exit the Program."
+            )
             formatting()
-            print("Program exited successfully.")
-            break
-        elif choice == 1:
-            list_books()
-        elif choice == 2:
-            add_book()
-        elif choice == 3:
-            search_book()
-        elif choice == 4:
-            borrow_book()
-        elif choice == 5:
-            return_book()
-        elif choice == 6:
-            remove_book()
+            dumb_choice = True
```

Comment:
Please refrain from offensive language. Change it to `invalid_choice`
```suggestion
            invalid_choice = True
```